astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -817,21 +816,6 @@ def _get_ecsv_header_dict(text):
     return yaml.safe_load("\n".join(lines))
 
 
-def _make_expected_values(cols):
```

Comment:
I haven't dug into it, but this function was likely used to generate some test data and might be needed again. In any case removing this function is out of scope for this PR.

Reply:
(reverted)